File: AtCoder/ABC/ABC101-150/abc126/abc126b.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int s;
  cin >> s;
  if(1<=s/100 && s/100<=12 && 1<=s%100 && s%100<=12) puts("AMBIGUOUS");
  else if(1<=s/100 && s/100<=12) puts("MMYY");
  else if(1<=s%100 && s%100<=12) puts("YYMM");
  else puts("NA");
  return 0;
}

File: AtCoder/ABC/ABC101-150/abc126/abc126b.py
s = input()
m = [i for i in range(1, 13)]
if int(s[2:]) in m and int(s[0:2]) in m:
  print("AMBIGUOUS")
elif int(s[2:]) in m:
  print("YYMM")
elif int(s[0:2]) in m:
  print("MMYY")
else:
  print("NA")